src/components/Person/Person.jsx

```diff
@@ -1 +1,12 @@
-// export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (<p className="Person__age">{person.age && `I am ${person.age}`}</p>)}
+    <p className="Person__partner">
+      {person.isMarried ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}` : `I am not married`}
```

Comment:
Consider will be better if you move this logic to helper variable and use it here
${person.sex === 'f' ? 'husband' : 'wife'}` : `I am not married`

Reply:
still not fixed this comment